example/demo/views.py

```diff
@@ -102,3 +108,18 @@ def post(self, request, *args, **kwargs):
             request=request,
         )
         return redirect("chat_thread", thread_id=thread_id)
+
+
+class TourGuideAssistantView(View):
+    def get(self, request, *args, **kwargs):
+        coordinates = request.GET.get("coordinate")
+
+        if not coordinates:
+            return JsonResponse({})
+
+        thread = create_thread(name="Tour Guide Chat", user=User.objects.first())
```

Comment:
Can you add current dt in isoformat here to generate unique names for threads?
Also, we shouldn't use the User.objects.first, but the current user instead.

Reply:
Missed that one there, thanks